README.md

```diff
@@ -33,7 +33,7 @@ You can find documentation at [www.chartjs.org/docs](http://www.chartjs.org/docs
 
 ## Contributing
 
-Before submitting an issue or a pull request, please take a moment to look over the [contributing guidelines](https://github.com/chartjs/Chart.js/blob/master/CONTRIBUTING.md) first. For support using Chart.js, please post questions with the [`chartjs` tag on Stack Overflow](http://stackoverflow.com/questions/tagged/chartjs).
+Before submitting an issue or a pull request, please take a moment to look over the [contributing guidelines](http://www.chartjs.org/docs/#notes-contributing) first. For support using Chart.js, please post questions with the [`chartjs` tag on Stack Overflow](http://stackoverflow.com/questions/tagged/chartjs).
```

Comment:
Same here, link to: `/chartjs/Chart.js/blob/master/docs/developers/contributing.md` but  without `https://github.com`.